src/runtime/ndarray.cc

```diff
@@ -159,7 +164,8 @@ NDArray NDArray::EmptyShared(
     const std::string& name, std::vector<int64_t> shape, DGLDataType dtype,
     DGLContext ctx, bool is_create) {
   NDArray ret = Internal::Create(shape, dtype, ctx);
-  // setup memory content
+  // setup memory content (i.e., allocation), other infos are setup in
+  // Internal::Create
```

Comment:
What does "other info" refer to?

Reply:
Sorry for the ambiguity.  I mean other infos like strides, shape, etc, have been set up in Internal::Create already, so only the size is assigned below. 
Updated.